datasources/build.conf

```diff
@@ -23,7 +23,7 @@ in <a href="/secure/administration/dataSourceManagement">the dedicated managemen
 # - x.y(.z): plugin major.minor.micro. Micro should be omitted. When omitted, z is assumed to be 0. 
 # For the build, we split the information between two properties, rudder branch and plugin version, 
 # which must be concaneted with "-" to build the plugin version. 
-plugin-branch=1.5-SNAPSHOT
+plugin-branch=1.4.1-SNAPSHOT
```

Comment:
Why do you change this version we should do it in 1.5, we have already some fixes that are merged for 4.3-1.5